src/rt/sections_darwin_64.d

```diff
@@ -176,3 +178,28 @@ const(section_64)[] sections(const segment_command_64* segment) pure nothrow @no
     const firstSection = cast(const(section_64)*)(cast(ubyte*) segment + size);
     return firstSection[0 .. segment.nsects];
 }
+
+/// Identifies a section via segment and section names.
+struct SectionRef
+{
+    immutable(char)* seg;
+    immutable(char)* sect;
+}
```

Comment:
Personally I'd rather have them as string, as previously. Why the change ?

Reply:
Because the single usage - for `getSection()` below - requires null-terminated char* pointers.

Reply:
I've refactored this & amended the last commit.